python/ray/air/config.py

```diff
@@ -326,6 +330,9 @@ class DatasetConfig:
     def __repr__(self):
         return _repr_dataclass(self)
 
+    def _repr_html_(self) -> str:
+        return make_table_html_repr(self)
```

Comment:
Nit: any reason why this and the other configs don't have titles? Thought it looked weird that only the "Backend Config" tab had a title.

Reply:
No reason we couldn't add it. In this instance I kind of felt it was redundant, as the tab that displays this HTML repr is already called "Dataset Config". In contrast, there are several different types of backend configs that could be used here (e.g. `TorchConfig`, `HorovodConfig`, `TensorflowConfig`, etc), so I felt a title was more appropriate in that case.